tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam/precice-config.xml

```diff
@@ -1 +1 @@
-precice-config_serial.xml
\ No newline at end of file
+nearest-projection-setup.xml
```

Comment:
Again, by moving the tutorial to another directory we can solve issues such as this one. ;-)

Reply:
I will make the tutorial case now as similar as possible to the flow-over-plate.xml tutorial.